<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <title>Document</title>
  <script>  //p.372 예외처리 구문에서 return 사용하기(1)
    function test () {
      try {
        alert('A 위치입니다.')
        throw "예외 강제발생"
      } catch (exception) {
        alert('B 위치입니다.')
        return
      }
      alert('C 위치입니다.')
    }

    //함수를 호출합니다.
    test()
  </script>
</head>
<body>
  
</body>
</html>